### ISSUE TITLE
Several issues raised by Clement

### ISSUE THREAD

**Issue by nikswamy** [FStarLang/FStar#341]
-- Unicode in strings sent over a pipe in the interactive mode ... b64 encode?
-- Sort the options
-- Error format: split errors without knowing where the next error starts. So, continuation lines are indented.
-- Strange error from the interactive mode when you send just a comment (cannot check more that one module or some variant thereof)
-- Some kind of info construct to get more semantic info from the compiler
-- So, maybe info messages are a different class of message, but formatted exactly like errors and warnings
-- test cases that report a mixture of errors and warnings

**Comment by cpitclaudel** [comment]
By the way: Am I correct in assuming that Emacs doesn't actually need to parse build config until it has support for interactive checking?

At the moment, since it just calls F* on one file, the build config will be parsed by F* itself, correct?

**Comment by cpitclaudel** [comment]
I've just pushed support for `(***`, `(**`, `(*!`, and `(*+` comments.

**Comment by cpitclaudel** [comment]
Regarding the server protocol: Dafny uses a server model documented at https://github.com/boogie-org/boogie-friends/blob/master/emacs/inferior-dafny.el#L62

**Comment by nikswamy** [comment]
Yes

Sent from my Windows Phone
________________________________

**Comment by catalin-hritcu** [comment]
@cpitclaudel Is anything here still an issue now?

**Comment by cpitclaudel** [comment]
Info messages were added recently.  Test cases would still be nice, but I think I understand enough F* to make some myself now :) Sending just a comment is still an error, but I opened a separate issue for that. I found a trick to parse errors without indented continuation lines, too.

TL;DR: let's close this :)
